### DISCUSSION TITLE
Barcamp on "AI in Publication Workflows" at Open Access Tage 2024

### DISCUSSION THREAD

**Discussion by xldrkp** [singlesourcepub/community#58] [Ideas]
In today's meeting we thought about a BarCamp session at [Open Access Tage 2024](https://open-access-tage.de/open-access-tage) to talk about the potential of AI in publication workflows. No ambitions, just thinking big and sharing ideas. Anybody interested? 

Next community call to discuss this and other things is Feb 1st 2024, 12:00 pm.

**Comment by castedo** [comment]
I will try to make it. Folks might be interested in looking at and hearing about the AI based copyediting tool I use in my "self-publication" workflow. It is documented at https://copyaid.it/ I've done a decent amount of systematic testing of prompts for the task of copyediting and have definitely formed some opinions and hot takes I'm happy to share.

The systematic prompt testing is not documented, but for brave souls, the source code is at https://gitlab.com/castedo/copyblaster, my main test data is at https://gitlab.com/castedo/copyblast and another partial example of test data is at https://github.com/castedo/manublast
